logistics_planning_base/models/logistics_schedule.py

```diff
@@ -295,6 +298,13 @@ def action_logistics_schedule_finished(self):
     def action_logistics_schedule_done(self):
         self.browse(self.env.context.get("active_ids", []))._action_done()
 
+    def _action_draft(self):
+        to_draft = self.filtered(lambda x: x.state in ['ready', 'done', 'cancel'])
```

Comment:
Desired flow for `draft` state as destination is only from `cancel` state:
```suggestion
        to_draft = self.filtered(lambda x: x.state == 'cancel')
```